data/data.go

```diff
@@ -14,7 +14,7 @@ import (
 	"regexp"
 	"strings"
 
-	effx_api "github.com/effxhq/effx-api-v2/generated/go/client"
+	effx_api "github.com/effxhq/effx-api-v2-go/generated/go/client"
```

Comment:
does it have to be nested? could it just be `github.com/effxhq/effx-api-v2-go/client` ?

Reply:
a good idea, no reason. I un-nested it.